### ISSUE TITLE
Reporting MSE Loss value

### ISSUE THREAD

**Issue by code-Assasin** [SullyChen/Autopilot-TensorFlow#33]
Hey thanks a lot for opensourcing your code and dataset. With a mse loss i know it's difficult to compare accuracy but could you report the mse loss you achieved(on an average) on your validation set?
